plugins/woocommerce-blocks/assets/js/base/components/cart-checkout/totals/coupon/index.tsx

```diff
@@ -122,12 +121,12 @@ export const TotalsCoupon = ( {
 					</form>
 					<ValidationInputError
 						propertyName="coupon"
-						elementId={ textInputId }
+						elementId={ instanceId }
 					/>
 				</div>
 			</LoadingMask>
 		</Panel>
 	);
 };
 
-export default withInstanceId( TotalsCoupon );
+export default TotalsCoupon;
```

Comment:
The `withInstanceId` HOC should no longer wrap the `TotalsCoupon` component, so the `instanceId` prop can be manually passed in.